tests/console/sysstat.pm

```diff
@@ -72,7 +72,7 @@ sub run {
     validate_script_output "mpstat",     sub { /CPU    %usr   %nice    %sys %iowait    %irq   %soft  %steal  %guest  %gnice   %idle/ };
     validate_script_output "sar -u",     sub { /CPU     %user     %nice   %system   %iowait    %steal     %idle/ };
     validate_script_output "sar -n DEV", sub { /IFACE   rxpck\/s   txpck\/s    rxkB\/s    txkB\/s   rxcmp\/s   txcmp\/s  rxmcst\/s   %ifutil/ };
-    if (is_sle('<=15-SP2') || is_leap('<=15.2')) {
+    if (is_sle('<=15-SP3') || is_leap('<=15.3')) {
```

Comment:
Feels a bit that we would be fine with just `is_sle || is_leap` condition. WDYT?

Reply:
I actually expect that I might have to change it again when the changes from TW hit the SP3 project.
It seems that the TW version is just newer and has this different columns.

Reply:
Right, ok, let's merge it then. What we could do is to parse version of the `sar` package itself, so it would be distribution independent and won't require any changes once we get it in the build.